e2e/test_android/login/login_spec.js

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// *********************************************************************
+// - [#] indicates a test step (e.g. # Go to a screen)
```

Comment:
Not necessary. However, this guide is especially important for non-tech QA to verify (by reading in English) whether the script (thru comments) follows what is written in the release test spreadsheet. It may not be one-to-one but at least they understand what's going on without understanding the code.
We have this style guide on Cypress, and by putting it on top of each file makes everyone aware to (strictly) follow and be guided without constant reminders. Open to suggestions.